identify/extensions.py

```diff
@@ -206,6 +206,7 @@
     '.yamllint': EXTENSIONS['yaml'] | {'yamllint'},
     '.zshrc': EXTENSIONS['zsh'],
     'AUTHORS': EXTENSIONS['txt'],
+    '.BUILD': {'text', 'bazel'},
```

Comment:
this mapping is for conventional names, the extensions mapping is above -- I don't think this patch does what you want 🤔

Reply:
or, maybe I'm missing the intention here -- maybe you could add a test or two to demonstrate this :)

Reply:
To be honest, I don't understand at all how this repo works - this is merely pattern matching on my part. My intent is to detect files ending in `.BUILD` and classify them the same way as `BUILD.bazel` or `BUILD` files.

Reply:
What would be the best way to get such a change added?

Reply:
I would suggest writing a test demonstrating what you want to do, and then adjusting this value until the test passes -- I'm not sure this is the correct place or value for what you're trying to do and a test should help demonstrate that.

I think the first mapping is more appropriate, but I also think it does case-insensitive matching on extensions so it's maybe not quite a perfect fit.  It's possible that what you have written is correct, but it would be better from a pull request perspective to demonstrate that precisely